client/rpc/rpcclient.go

```diff
@@ -1,8 +1,146 @@
 package rpcclient
 
-import "github.com/btcsuite/btcd/rpcclient"
+import (
+	"github.com/btcsuite/btcd/btcjson"
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/rpcclient"
+	"github.com/btcsuite/btcd/wire"
+)
 
 // RPCClient represents a JSON RPC connection to a bitcoin node.
 type RPCClient struct {
 	*rpcclient.Client
 }
+
+func New(config *rpcclient.ConnConfig, ntfnHandlers *rpcclient.NotificationHandlers) (*RPCClient, error) {
+	internal_client, err := rpcclient.NewBatch(config)
+	if err != nil {
+		return nil, err
+	}
+	client := RPCClient{
+		internal_client,
+	}
+	return &client, nil
+}
+
+// Helper function to batch translating block numbers to hashes
```

Comment:
It is convention in Go start comments with the name of the function, but if this is just a helper since it is not part of the Client interface then I say just make it private.

Reply:
Same with the `GetBlocksByHashes`, `GetBlocksVerboseByHashes`, and `GetBlocksVerboseTxByHashes`

Reply:
Gotcha. I'll just make them private for now and we can make them public + add comments later if we end up wanting them in the interface.